.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
We maybe should exclude png and nc files and other binaries from text 

https://github.com/desktop/desktop/issues/5323#issuecomment-410220449

We don't have  text eol=lf but it maybe useful it did not lookup binaries.

Reply:
`* text=auto` tells git to auto-detect if a file is text or binary, for all files in the repository. Does it wrongly detect a png or nc as text somewhere?

The issue you linked is solely caused by `* text`, which tells git that it should consider all files to be text files.

Reply:
I tried in the normalize-files branch

```
files_with_crlf="$(git ls-files --eol)"
echo "$files_with_crlf"
```
current .gitattributes
```
i/-text w/-text attr/text=auto          tutorials/pictures/wms/win/selecttoopencontrol.png
i/-text w/-text attr/text=auto          tutorials/pictures/wms/win/star_filter.png
```

with *.png -text
```
i/-text w/-text attr/-text              tutorials/pictures/wms/win/selecttoopencontrol.png
i/-text w/-text attr/-text              tutorials/pictures/wms/win/star_filter.png
```
This changes the eloattr

Reply:
Yes, since that is what .gitattributes does, changing the files attributes. `* text=auto` tells git to auto-detect if a file is text or binary and as seen in `i/-text w/-text` it successfully did so. With `text=auto` it detected those png files as "not text" (i.e. binary) in the index and in the working tree.